### PR TITLE
feature(resources): Add syntax highlighting for languages used in articles

### PR DESCRIPTION
**Pull request by supergrecko**
This PR adds syntax highlighting support for the languages already used in the resources. It also covers the languages which are requesting resources (Java, C++/C)

### CODE REVIEW

**Comment by veksen** [comment]
This doesn't build at the moment, I gotta look into this.

**Comment by veksen** [comment]
Build error possibly resolved by https://github.com/the-programmers-hangout/website/pull/123

File: src/components/Markdown/index.tsx [hunk 1]
```diff
@@ -15,9 +19,9 @@ interface IMarkdownProps {
   mainly to provide styles to the html generated.
 */
 export const Markdown: FC<IMarkdownProps> = ({
-  content,
-  ...restProps
-}) => {
+                                               content,
+                                               ...restProps
+                                             }) => {
```
Comment (veksen) [inline src/components/Markdown/index.tsx]: You can add Prettier on save in your editor, or run `npm run format` :)

Comment (supergrecko) [reply src/components/Markdown/index.tsx]: Yeah, no worries, I accidently copy pasted my branch which wasn't rebased so it pushed those changes. Undid it. should be good now

**Comment by veksen** [comment]
perfect!

**Pull request merged**